src/main/java/com/mixpanel/android/viewcrawler/EditState.java

```diff
@@ -159,7 +171,12 @@ public void run() {
             }
             // ELSE View is alive and we are alive
 
-            mEdit.visit(viewRoot);
+            try {
+                mEdit.visit(viewRoot);
+            } catch (ViewVisitor.LayoutUpdateException e) {
+                cleanUp();
+                throw e;
```

Comment:
Sorry, I think I was wrong about that. The exception will have to be caught above.